.github/workflows/review-app-creation.yml

```
@@ -22,7 +22,8 @@ env:
 jobs:
   create:
     runs-on: ubuntu-latest
-    permissions: {}
+    permissions:
+      issues: write
```

```suggestion
      pull-requests: write
```

Contrairement à ce que la doc semble dire, dans notre cas on  ne peut pas trop intervertir `issues` et `pull-requests`.

Le test est là : https://github.com/gip-inclusion/les-emplois/actions/runs/14731189445/job/41345626492?pr=5988
(commit https://github.com/gip-inclusion/les-emplois/pull/5988/commits/737953090c40259d1b05343bfdfb4b8c1eaab8d6)